Agent/Linux/ubuntu1804-README.md

```diff
@@ -0,0 +1,44 @@
+# Azure DevOps - Ubuntu 18.04 Agent
+
+The following sections provide a step-by-step guide to creating a Azure DevOps Agent Image using Packer, spin up an Agent using the Image and add it to the Azure DevOps Agent Pool.
+
+## Prepare your Azure DevOps Account
+
+1. Create an Azure DevOps Agent pool.
```

Comment:
Is there a step by step guide on how to create an agent pool? Perhaps from the Azure DevOps doc site? Step 1 here  involves different steps.

Reply:
Added a link to the documentation for creating an Azure DevOps Agent pool.